src/main/resources/messages.properties

```
@@ -28,20 +30,24 @@ general.eligibility-table.row-5.family=6 people
 general.eligibility-table.row-5.income=$6,973
 general.button.continue=Continue
 
-general.week.monday=Monday
-general.week.tuesday=Tuesday
-general.week.wednesday=Wednesday
-general.week.thursday=Thursday
-general.week.friday=Friday
-general.week.saturday=Saturday
-general.week.sunday=Sunday
+general.week.Monday=Monday
+general.week.Tuesday=Tuesday
+general.week.Wednesday=Wednesday
+general.week.Thursday=Thursday
+general.week.Friday=Friday
+general.week.Saturday=Saturday
+general.week.Sunday=Sunday
+general.week.All=All days
 #
 errors.required=Make sure you answer this question.
 errors.provide-first-name=Make sure to provide a first name.
 errors.provide-last-name=Make sure to provide a last name.
 errors.provide-birthday=Make sure to provide a birthday.
 errors.date-outside-of-supported-range=Please check the date entered. {0} is not a supported start date.
-errors.invalid-date-entered=Please check the date entered. It is not a valid date.
+errors.invalid-past-date-entered=Please enter the date your child started care in this format: mm/dd/yyyy
+errors.invalid-future-date-entered=Please enter the date your child will start care in this format: mm/dd/yyyy
+errors.future-childcare-date-outside-of-range=Past date entered for future child care start date: Please choose a future start date
+errors.past-childcare-date-out-of-range=Past child care start date outside range 01/01/1901 - today: Please choose a start date between 01/01/1901 and today
```

I think we can get rid of the part before the colon here too
```suggestion
errors.future-childcare-date-outside-of-range=Please choose a future start date
errors.past-childcare-date-out-of-range=Please choose a start date between 01/01/1901 and today
```